docs/src/examples/components/Tooltip/Types/TooltipExample.shorthand.steps.ts

```diff
@@ -0,0 +1,11 @@
+import { Button } from '@stardust-ui/react'
+
+const config: ScreenerTestsConfig = {
+  themes: ['teams', 'teamsDark', 'teamsHighContrast'],
+  steps: [
+    builder => builder.hover(`.${Button.className}`).snapshot('Shows tooltip'),
+    builder => builder.hover('.ui-provider__box').snapshot('Shows tooltip'),
```

Comment:
What this step does? Naming is wrong? 🤔

Reply:
It's for resetting, we don't even need screenshot, will refactor it.